src/main/java/eutros/multiblocktweaker/crafttweaker/construction/RecipeMapBuilder.java

```diff
@@ -272,6 +288,11 @@ public RecipeMap<?> build() {
             map.setProgressBar(progressBarTexture, moveType);
         }
 
+        if (sound != null) {
+            if(ForgeRegistries.SOUND_EVENTS.containsKey(sound))
+                map.setSound(ForgeRegistries.SOUND_EVENTS.getValue(sound));
+        }
```

Comment:
It might be nice to log a warning if the resource isn't found.

Reply:
Done.